server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -105,6 +105,16 @@ it is around the label */
   @include u-height('105');
 }
 
+// Between sections
```

Comment:
**suggestion (non-blocking):** Is this just for sections in the block page or for sections everywhere in the app?  If it's just for the block view, would you mind adding that to the comment?

Reply:
It'll be used throughout the app